src/mono/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.Mono.cs

```diff
@@ -32,8 +32,20 @@ public static void InitializeArray(Array array, RuntimeFieldHandle fldHandle)
         [Obsolete("OffsetToStringData has been deprecated. Use string.GetPinnableReference() instead.")]
         public static int OffsetToStringData
         {
-            [Intrinsic]
-            get => OffsetToStringData;
+            [NonVersionable]
+            get =>
+                // Number of bytes from the address pointed to by a reference to
+                // a String to the first 16-bit character in the String.  Skip
+                // over the MethodTable pointer, & String
+                // length.  Of course, the String reference points to the memory
+                // after the sync block, so don't count that.
+                // This property allows C#'s fixed statement to work on Strings.
+                // On 64 bit platforms, this should be 12 (8+4) and on 32 bit 8 (4+4).
```

Comment:
Is this comment still correct  with 12/8 bytes ?

Reply:
fixed the comment, thanks!